src/screens/Auth/Register.tsx

```diff
@@ -13,6 +13,12 @@ import {
 import { supabase } from '../../utils/supabaseClient';
 import { AuthStackParamList } from './AuthNavigator';
 
+// new
```

Comment:
You can get rid of these comments like `// new`  `// background color` `//new, hide header` etc... github will show us what the changes are when you make a PR, and the background color comment is redundant